src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -38,6 +41,7 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
     getTopicDetail,
     onClose
   } = props;
+  const [, discoveryDispatch] = (React.useContext(Context) as any)?.discovery;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `as any` is generally discouraged as it bypasses TypeScript's type checking. It would be better to define a proper type for the context. If the context structure is complex, consider creating an interface or type for it.
```diff
- const [, discoveryDispatch] = (React.useContext(Context) as any)?.discovery;
+ const [, discoveryDispatch] = React.useContext<YourDefinedContextType>(Context)?.discovery;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->